libs/contrib/Data/Void.idr

```diff
@@ -0,0 +1,13 @@
+module Data.Void
+
+export
+persistentVoid : (0 x : Void) -> Void
+persistentVoid _ impossible
+
+export
+absurdity : Uninhabited t => (0 _ : t) -> s
+absurdity x = absurd (persistentVoid (uninhabited x))
```

Comment:
This is `void (uninhabited x)`

Reply:
I know it's petty, but it's annoying that we have `absurd` but write `void (uninhabited)` when we're irrelevant.
I propose we either keep `absurdity`, or change `absurd` to take an irrelevant argument. 
(I'd prefer the latter, but that would mean changing `base`.)

Reply:
I think the problem with that proposal is that `No absurd` will
not typecheck anymore because `No` expects a proof-relevant
function. So you'd need to manually go expand all of the existing
`No absurd` to `No (\ 0 x => absurd x)`.

Reply:
I'll keep absurdity then.